tests/performance/locust_files/.env-devel

```diff
@@ -0,0 +1,11 @@
+# user
+SC_USER_NAME=
+SC_PASSWORD=
+
+# locust settings
+LOCUST_HOST=http://10.43.103.193.nip.io:8005
```

Comment:
rm this ip... add localhost or something equivalent

Reply:
I will use the convention of doing `replace-with-<name>` for all the fields the user must/should add.